src/node_contextify.cc

```diff
@@ -110,9 +110,28 @@ using v8::Value;
 
 namespace {
 
+// Utility function to get the file extension from a filename
+std::string GetFileExtension(std::string_view filename) {
+  size_t dot_position = filename.find_last_of('.');
+  if (dot_position == std::string_view::npos) {
+    return "";  // No extension found
+  }
+  return std::string(filename.substr(dot_position));
+}
+
+std::string GetWarningMessage(const std::string& file_extension) {
+  if (file_extension == ".cjs") {
+    return "Cannot use 'import' statement in a .cjs file. "
```

Comment:
This could be also caused by e.g. top-level await. It would be confusing to directly suggest import as the cause of the parse error without checking the error messages (which are available in as `cjs_message` in `CompileFunctionForCJSLoader`).